lib/issues.js

```diff
@@ -31,21 +31,24 @@ module.exports.insertIssue = async ({
   systemData,
   authnUserId,
 }) => {
-  // Truncate all strings in `courseData` to 1000 characters. This ensures
+  // Truncate all strings in the data objects to 1000 characters. This ensures
   // that we don't store too much unnecessary data. This data is here for
   // convenience, but it's not the source of truth: pretty much all of it
   // is also stored elsewhere in the database, so we can always retrieve it
   // if needed. The worst data is submission data, which can be very large;
   // this is stored on each individual submission.
   const truncatedCourseData = recursivelyTruncateStrings(courseData, 1000);
+  // Allow for a higher limit on the system data. This object contains output
+  // from the Python subprocess, which can be especially useful for debugging.
+  const truncatedSystemData = recursivelyTruncateStrings(systemData, 10000);
```

Comment:
I picked 10000 bytes somewhat at random; open to suggestions for other limits!

Reply:
I don't want to read more than 10k of debug output 😅